stream-chat-android-state/src/main/java/io/getstream/chat/android/state/event/handler/internal/utils/ChatEventUtils.kt

```diff
@@ -44,8 +44,7 @@ internal fun NotificationMarkReadEvent.toChannelUserRead() = ChannelUserRead(
     lastRead = createdAt,
 // TODO: remove this once the backend is fixed and is sending us the number of unread messages
     unreadMessages = 0,
-// TODO: Backend should send us the last read message id
-    lastReadMessageId = null,
+    lastReadMessageId = lastReadMessageId,
```

Comment:
Is `lastReadMessageId` the right one?
I think in the past it wasn't a valid one and we had some issues with `markUnread` feature.
This `lastReadMessageId` is used within our MessageListControler to find the position, if any, where the "Unread Messages" label needs to be shown.